react/week1/todo-app/src/activities.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+function HeaderTitle() {
+  return (
+    <>
```

Comment:
you don't actually need the fragments here

Reply:
done!